### PR TITLE
Release v0.2.4

### PR DESCRIPTION
**Pull request by digicosmos86**
We can merge this and publish the release once we have the `.dic()` method

### CODE REVIEW

**Review by cpaniaguam — COMMENTED** [review]
It might be a good idea to put the linting and formatting from the rest of the functional tests.

**Comment by cpaniaguam** [comment]
> Not sure what you mean? Do you mean in ci?

Yes, in CI. That way in the report you can quickly see what went wrong without having to check the detailed error messages. It'd have the added benefit the of having the tests run independently of each other.

File: pyproject.toml [hunk 1]
```diff
@@ -17,31 +17,32 @@ keywords = ["HSSM", "sequential sampling models", "bayesian", "bayes", "mcmc"]
...
 ssm-simulators = "^0.7.2"
-huggingface-hub = "^0.23.0"
+huggingface-hub = "^0.24.6"
 bambi = ">=0.14.0,<0.15.0"
-numpyro = "^0.15.0"
+numpyro = "^0.15.2"
 hddm-wfpt = "^0.1.4"
 seaborn = "^0.13.2"
 jax = { version = "^0.4.25", extras = ["cuda12"], optional = true }
+numpy = ">=1.26.4,<2.0.0"
```
Comment (cpaniaguam) [inline pyproject.toml:29]: I guess it did the trick!

**Review by cpaniaguam — APPROVED** [review]
Mischievous numpy 2.0!

**Comment by digicosmos86** [comment]
> Mischievous numpy 2.0!

Looks like it! Thanks Carlos!

**Pull request merged**